server/views/applications/pages/oasys-import/rosh-summary.njk

```diff
@@ -0,0 +1,122 @@
+
+{% set columnClasses = "govuk-grid-column-full" %}
+
+{% block questions %}
+  <div class="govuk-grid-row">
+    <div>
+      <h1 class="govuk-heading-l">{{page.title}}</h1>
+      <p>Imported from OASys 23 October 2022</p>
+    </div>
+
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        attributes: {
+          'role': 'tablist'
+        },
+        items: [{
+          text: 'RoSH summary',
+          href: '#roshSummary',
+          attributes: {
+            'aria-controls': 'roshSummary',
+            'id': 'roshSummaryTab',
+            role: 'tab',
+            'aria-selected': true
+          }
+        }, 
+        {
+          text: 'Offence details',
+          href: '#offenceDetails',
+          attributes: {
+            'aria-controls': 'offenceDetails',
+            'id': 'offenceDetails',
+            role: 'tab'
+          }
+        }, 
+        {
+          text: 'Supporting information',
+          href: '#supportingInformation',
+          attributes: {
+            'aria-controls': 'supportingInformation',
+            'id': 'supportingInformation',
+            role: 'tab'
+          }
+        },  
+        {
+          text: 'Risk management plan',
+          href: '#riskManagementPlan',
+          attributes: {
+            'aria-controls': 'riskManagementPlan',
+            'id': 'riskManagementPlan',
+            role: 'tab'
+          }
+        },
+        {
+          text: 'Risk to self',
+          href: '#riskToSelf',
+          attributes: {
+            'aria-controls': 'riskToSelf',
+            'id': 'riskToSelf',
+            role: 'tab'
+          }
+        }]
+      }) }}
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds" id="caseNotes" role="tabpanel" aria-labelledby="caseNotesTab" hidden>
+      {{page.roshTextAreas() | safe }}
+
+    </div>
+    <div class="govuk-grid-column-one-third">
+      {{ widgets(page.risks) }}
+    </div>
+  </div>
+
+{% endblock %}
+
+{% block extraScripts %}
+  <script type="text/javascript" nonce="{{ cspNonce }}">
+    function hidePanels() {
+      for (var i = 0; i < panels.length; i++) {
+        panels[i].setAttribute('hidden', 'true')
+      }
+    }
+
+    function showPanel(panel) {
+      hidePanels()
+      panel.removeAttribute('hidden')
+    }
+
+    function setCurrent(current) {
+      for (var i = 0; i < panels.length; i++) {
+        navLinks[i].removeAttribute('aria-selected')
+      }
+
+      current.setAttribute('aria-selected', 'true')
+    }
+
+    var navLinks = document.querySelectorAll('[role="tab"]')
+    var panels = document.querySelectorAll('[role="tabpanel"')
+
+    showPanel(panels[0])
+
+    for (var i = 0; i < navLinks.length; i++) {
+      navLinks[i].addEventListener('click', function (e) {
+        var link = e.target
+        var id = link
+          .attributes['aria-controls']
+          .value
+        var panel = document.querySelector('#' + id)
```

Comment:
Could getElementById be used here?

Reply:
This was copied from `case-notes.njk` but we're going to be using some different mechanism to change tabs so I have removed the code for now